### PR TITLE
transparent-validation/twonkie/introduction.md: add Twonkie documentation

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/transparent-validation/twonkie/introduction.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+1. Verify in `lsusb` that the device has booted into firmware:
+
+   ```bash
+   $ lsusb | grep 18d1:500a
+   Bus 003 Device 084: ID 18d1:500a Google Inc. Twinkie
+   ```
+
+### Testing
+
+Sniffing a PD negotiation:
```
Comment (Psotas) [inline docs/transparent-validation/twonkie/introduction.md:78]:
```suggestion
#### Sniffing a PD negotiation:
```

File: docs/transparent-validation/twonkie/introduction.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+   ```bash
+   $ sudo minicom -D /dev/ttyUSB0
+   ```
+
+1. Type in `tw trace on` to start logging
+1. Connect a twonkie between a USB-PD power supply and a USB-PD power sink
+1. Verify that the device is charging and logs appear on the screen
+
+Measuring voltage and current:
```
Comment (Psotas) [inline docs/transparent-validation/twonkie/introduction.md:90]:
```suggestion
#### Measuring voltage and current:
```

File: docs/transparent-validation/twonkie/introduction.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+1. Open minicom on Twonkie's console:
+
+   ```bash
+   $ sudo minicom -D /dev/ttyUSB0
+   ```
+
+1. Connect a twonkie between a USB-PD power supply and a USB-PD power sink
+1. Type in `tw vbus` to display the voltage and current measurement
+
+Power sink:
```
Comment (Psotas) [inline docs/transparent-validation/twonkie/introduction.md:101]:
```suggestion
#### Power sink:
```

**Review by Psotas — COMMENTED** [review]
Add heading level 4

File: docs/transparent-validation/twonkie/introduction.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# Introduction
```
Comment (Psotas) [inline docs/transparent-validation/twonkie/introduction.md:1]:
```suggestion
# Twonkie Introduction
```

Comment (mkopec) [reply docs/transparent-validation/twonkie/introduction.md]: fixed in fbc6fee1fbccdbb775299550ebc3d659a5049268

File: docs/transparent-validation/twonkie/introduction.md (hunk 3 above)
Comment (mkopec) [reply docs/transparent-validation/twonkie/introduction.md]: fixed in fbc6fee1fbccdbb775299550ebc3d659a5049268

File: docs/transparent-validation/twonkie/introduction.md (hunk 2 above)
Comment (mkopec) [reply docs/transparent-validation/twonkie/introduction.md]: fixed in fbc6fee1fbccdbb775299550ebc3d659a5049268

File: docs/transparent-validation/twonkie/introduction.md (hunk 1 above)
Comment (mkopec) [reply docs/transparent-validation/twonkie/introduction.md]: fixed in fbc6fee1fbccdbb775299550ebc3d659a5049268

**Comment by mkopec** [comment]
@Psotas added headings as per review

**Comment by mkopec** [comment]
@BeataZdunczyk Added photos per your request.

**Review by BeataZdunczyk — APPROVED** [review]

**Pull request merged**